internal/app/upgradeagent/container.go

```diff
@@ -46,3 +47,24 @@ func (c Container) ImageName() string {
 
 	return imageName
 }
+
+// Labels returns labels information for dedicated container
+func (c Container) Labels() map[string]string {
+	return c.containerInfo.Config.Labels
+}
+
+// GetProjectName returns project name for given container
+func (c Container) GetProjectName() (string, error) {
+	if name, found := c.containerInfo.Config.Labels[DockerComposeProjectLabel]; found {
+		return name, nil
+	}
+	return "", errors.New("can't get project name")
+}
```

Comment:
```suggestion
func (c Container) GetProjectName() (string, bool) {
	name, found := c.containerInfo.Config.Labels[DockerComposeProjectLabel]
	return name, found
}
```

I think that not finding the project.name is not an error. So we should just indicate whether a value was found or not.

Reply:
fixed.